### PR TITLE
Add Sample Rails App

### PR DESCRIPTION
**Pull request by hennevogel**
For poor people who don't have a Rails app. Also maybe for trying how things feel
during development ;-)

### CODE REVIEW

File: sample-dashboard/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+RUN zypper -n clean -a
+RUN gem install --no-format-executable --no-document rails -v '~> 6'
+
+USER rails
+WORKDIR /home/rails
+
+RUN rails new thingiverse;
+
+WORKDIR /home/rails/thingiverse
+RUN rails g scaffold thing name:string amount:integer
```
Comment (ChrisBr) [inline sample-dashboard/Dockerfile:18]: What is a thing 😄 I think naming is important 😄 

What about doing the famous Rails blog 😄 article title:string body:string

File: sample-dashboard/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+RUN echo 'gem "influxdb-rails", :git => "https://github.com/influxdata/influxdb-rails/", :branch => "master"' >> Gemfile
+RUN bundle install; rails webpacker:install; rake db:migrate
+RUN sed -i '2i \  root to: "things#index"' config/routes.rb
+RUN bundle exec rails generate influxdb
+RUN curl -o app/controllers/application_controller.rb https://gist.githubusercontent.com/hennevogel/296e93b9db620e9cf48eeb30cef6a145/raw/8c6d4635ded3bef2ca9cfdf65c5863cf6f03a984/application_controller.rb
```
Comment (ChrisBr) [inline sample-dashboard/Dockerfile:23]: That's a recipe for breaking, no?

Also wondering if this should not be part of the actual gem? Is there a reason somebody doesn't want to have this in the data? We could make this more reliant with checking if a request object exists and responds to id.

```
InfluxDB::Rails.current.values = { request: request.request_id }
```

Edit: I think we don't have this information in the gem 😢

**Review by ChrisBr — APPROVED** [review]

File: sample-dashboard/Dockerfile (hunk 1 above)
Comment (hennevogel) [reply sample-dashboard/Dockerfile:18]: How boring :wink:  Things are awesome! I have many of them.

Comment (hennevogel) [reply sample-dashboard/Dockerfile:18]: We could use some [template](https://guides.rubyonrails.org/rails_application_templates.html) but I don't know any...

File: sample-dashboard/Dockerfile (hunk 2 above)
Comment (hennevogel) [reply sample-dashboard/Dockerfile:23]: We don't...

**Pull request merged**